detection_rules/config.py

```diff
@@ -150,16 +151,39 @@ class RulesConfig:
     exception_dir: Optional[Path] = None
 
 
+def check_path(content: dict, keys: tuple, content_name: str):
+    """Check if a dict key path exists in the supplied content."""
+    try:
+        reduce(lambda d, key: d[key], keys, content)
+    except KeyError:
+        raise ValueError(f"Missing {'.'.join(keys)} in {content_name} contents.")
+
+
+def validate_package_contents(package_contents: dict):
+    """Validate the contents of the package for required fields."""
+
+    paths = [
+        ('name',),
+        ('registry_data', 'conditions'),
+        ('registry_data', 'conditions', 'kibana.version')
+    ]
+
+    for path in paths:
+        check_path(package_contents, path, "packages.yml")
+
+
 @cached
 def parse_rules_config(path: Optional[Path] = None) -> RulesConfig:
     """Parse the _config.yaml file for default or custom rules."""
     if path:
         assert path.exists(), f'rules config file does not exist: {path}'
         loaded = yaml.safe_load(path.read_text())
-    elif CUSTOM_RULES_DIR:
-        path = Path(CUSTOM_RULES_DIR) / '_config.yaml'
-        assert path.exists(), f'_config.yaml file missing in {CUSTOM_RULES_DIR}'
+    elif CUSTOM_RULES_DIR and (path := Path(CUSTOM_RULES_DIR) / '_config.yaml').exists():
         loaded = yaml.safe_load(path.read_text())
+        if 'configuration_details' in loaded or not loaded:
+            print(loaded['configuration_details'])
+            raise SystemExit(0)
```

Comment:
We can also combine these two lines if we really think this is an error. IMO it's more of a NOP than an error, hence the `0`.

Reply:
Sounds good to me, same functionality as `sys.exit` :+1: